.storybook/preview.js

```diff
@@ -1,14 +1,26 @@
 import { addDecorator, addParameters } from '@storybook/html';
 import { withA11y } from '@storybook/addon-a11y';
-import centered from '@storybook/addon-centered/html';
+import { withPaddings } from 'storybook-addon-paddings';
 import * as colors from '../src/design-tokens/colors.yml';
+import * as ms from '../src/design-tokens/modular-scale.yml';
```

Comment:
Just to see how this works (with whatever webpack loader I assume is responsible), I tried this

```js
import { ratio } from '../src/design-tokens/modular-scale.yml';
//...
value: `${Math.pow(ratio, idx).toFixed(2)}em`,
```
Seems to work. If the explicitness of `ms.ratio` is preferred, I'm not picky about which way it's imported.

Reply:
You're correct, the Theo loader is currently set up to write `export` statements for all of the design tokens in a file.

I've simplified in this case because I'm not sure what other values this value may export in the short term.